src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Locations/Queries/GetAddresses/GetAddressesQueryHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using MediatR;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.ApprenticeAan.Application.Locations.Queries.GetAddresses
+{
+    public class GetAddressesQueryHandler : IRequestHandler<GetAddressesQuery, GetAddressesQueryResult>
+    {
+        private readonly ILocationApiClient<LocationApiConfiguration> _locationApiClient;
+
+        public GetAddressesQueryHandler(ILocationApiClient<LocationApiConfiguration> locationApiClient) => _locationApiClient = locationApiClient;
+
+        public async Task<GetAddressesQueryResult> Handle(GetAddressesQuery request, CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrEmpty(request.Query)) throw new ArgumentException("Query is required", nameof(request));
+
+            var addressesResponse = await _locationApiClient.Get<GetAddressesListResponse>(new GetAddressesQueryRequest(request.Query, 0.1));
```

Comment:
`new GetAddressesQueryRequest(request.Query, 0.1));`

I've hardcoded `0.1` as the minMatch variable. Other handlers have that as an environment variable within their respective das-employer-config, but I've heard no requirement to do that for this project